configs/match.cfg

```diff
@@ -1,2 +1,23 @@
-@include compute/7gpu.cfg
-@include match-base.cfg
\ No newline at end of file
+@include ../engines/KataGo-raw/cpp/configs/match_example.cfg
```

Comment:
1. In match_example, resignation is allowed by default. Do we want to allow resignation in our `matches`, or have the bot play out all the way to the end?
2. Komi is also set to auto in match_example. Do we want to fix komi for our match games?

Reply:
1. I don't know, although I would prefer to stay as close to the default as possible, so absent a strong argument in favor of banning resignation I'd mildly prefer to keep it enabled.
2. I don't see a particular reason to set `komiAuto=false`.

Reply:
I'll look at the SGFs to see how many games are resigned

Reply:
Ah I guess with `komiAuto=true` I'm not sure what logic is used to determine komi. For example:
- `/nas/ucb/norabelrose/latest@1/7F37165C0E01B424.sgfs` has a komi of 17
- `/nas/ucb/norabelrose/latest@1/FDA39157AAE5AB5D.sgfs` has a komi of 15
- `/nas/ucb/norabelrose/latest@128-dragonslayer/C2A884B373B58DE3.sgfs` has a komi of 9
- `/nas/ucb/norabelrose/latest@1-dragonslayer/83A7BB83EF62755F.sgfs` has a komi of 9.5.

It's fine for us to vary the komi in `match`, but we should at least have a good idea of how the komi is set. I also lean towards us using a smaller more standard set of komi because those are what actual tournament matches are played with (e.g. 6.5 or 7.5 on a 9x9 board).

Reply:
As for resignation, I would lean towards turning it off by default for two reasons:
1. We get slightly more signal from our evaluation results in the form of win-margin.
2. Like Tom mentioned on Slack, winning with resignation turned off is more convincing just because it avoids any question of whether we tricked the victim into resigning (which could be easily patched).

Reply:
- I think the main benefit of resignation is saving time/compute, but I'm inclined to say we should do most of our evaluations with resignation disallowed — otherwise people may look at our games and wonder whether KataGo would win if we add the simple defense of disallowing resignation
- I feel suspicious of `komiAuto`. Could I make an adversary that estimated komi values that are super skewed towards white, and then the adversary would win 50% of the time just due to its komi estimates messing with `komiAuto`?

Reply:
Not sure exactly how `komiAuto` works, I looked at the code briefly and this is what I saw:
* `play.cpp`: `komiAuto` sets `extraBlackAndKomi.makeGameFairForEmptyBoard = true`. Then `PlayUtils::adjustKomiToEven()` is called to set komi automatically
* `playutils.cpp`: `adjustKomiToEven()` invokes `getNaiveEvenKomiHelper()` which does some kind of iterative search for a fair komi using `evalKomi()`. `evalKomi()` returns the `lead`/`winLoss` values at the root node after a search, and `getNaiveEvenKomiHelper()` finds a komi by looking for when the `lead`/`winLoss` values look fair

Reply:
I think we should definitely turn `komiAuto` off for evaluation, and set komi to 6.5 or 7.5 (either seems fine).

I can see an argument for and against resignation. I lean towards turning it off, but may be worth doing an ablation where we vary resignation along with other rule sets (Tromp-Taylor vs Chinese vs Japanese) to see how robust the results are.

Reply:
lightvector seems to think `komiAuto` is insane